tests-e2e/step-definitions/standard-ui/form.js

```diff
@@ -259,3 +259,13 @@ Then('The value of the Input field with id {string} should be {string}',
     await client.assert.value(`//input[@id='${id}']`, expected);
     client.useCss();
   });
+
+/**
+ * TODO: Checking only the first form input since looping and asserting
+ * in nightwatch is a real pain.
+ */
+Then('Each form field should be disabled',
```

Comment:
Maybe we should restrict the check for a formular title or id and we could achieve a valid check with the following:

```js
$x('//input[not(@disabled)]')
```

The selector will return all inputs, that are not disabled.